timelib.h

```diff
@@ -417,6 +482,19 @@ timelib_time *timelib_strtotime(char *s, size_t len, timelib_error_container **e
  */
 timelib_time *timelib_parse_from_format(char *format, char *s, size_t len, timelib_error_container **errors, const timelib_tzdb *tzdb, timelib_tz_get_wrapper tz_get_wrapper);
 
+/* Parses the date/time string in 's' with length 'len' into the constituent
+ * parts of timelib_time* according to the format in 'format' with format specifier
+ * configuration 'format_config'.
+ *
+ * 'format_map' is an array of pairs, with the first element being the format
+ * specifier as a character and the second element corresponds to the representation
+ * of the specifier from the enum list 'timelib_format_specifier_code'.
+ *
+ * Note: 'format_map' must be terminated with the pair {'\0', 0} to indicate to the
```

Comment:
``0`` or ``NULL`` ?

Reply:
Rewrote it to be clear, the second element of the pair actually doesn't matter.